lib/td/client/api/job.rb

```diff
@@ -165,7 +165,7 @@ def job_result_each(job_id, &block)
     upkr = MessagePack::Unpacker.new
     infl = nil
 
-    get("/v3/job/result/#{e job_id}", {'format'=>'msgpack'}) {|res, chunk, current_total_chunk_size|
+    get("/v3/job/result/#{e job_id}", {'format'=>'msgpack'}, {:resume => true}) {|res, chunk, current_total_chunk_size|
       if res.code != 200
```

Comment:
Hi, I'm trying this patch to resolve TreasureData::API::IncompleteError .
When resuming, the server responds with `206 Partial Content`, so `res.code == 206` is also acceptable here, I think.